<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{
color : #fff7f2;
border : none;
}
#centralwidget{
background-color: #191919;
}
#leftMenu{
background-color: #deb30f;
}
QLineEdit{
background : transparent;
}
#searchFrame{
border-radius :10px;
border : 2px solid #deb30f;
}
#appHeader{
color : #deb30f;
}
#card1, #card2,#card3, #card4 {
background-color : #191919; 
border-radius : 20px;
}

#widget_4,#widget_5,#profileCon{
background-color :  #191919; 
border-radius : 20px;
}
#pushButton_3{
background-color :  #191919; 
padding : 10px 5px;
text-align : left;
border-top-left-radius : 20px;
}
#pushButton_4,#pushButton_5,#pushButton_6,#pushButton_7,#pushButton_10,#pushButton_11,#pushButton, #pushButton_2{
padding : 10px 5px;
text-align : left;

}

#pushButton_10,#pushButton_11,#pushButton, #pushButton_2{
padding : 10px 10px;
text-align : center;

}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="MainBody" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignTop">
        <widget class="QWidget" name="headerFrame" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="appHeader">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Employees</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="cardsFrame" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7"/>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mainFrame" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="leftMargin">
                 <number>9</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_2">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <widget class="QTabWidget" name="tabWidget">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>1021</width>
                     <height>421</height>
                    </rect>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
                   </property>
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <widget class="QWidget" name="tab">
                    <attribute name="title">
                     <string>Add</string>
                    </attribute>
                    <widget class="QPushButton" name="pushButton">
                     <property name="geometry">
                      <rect>
                       <x>580</x>
                       <y>250</y>
                       <width>81</width>
                       <height>31</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>ADD</string>
                     </property>
                    </widget>
                    <widget class="QLineEdit" name="lineEdit_3">
                     <property name="geometry">
                      <rect>
                       <x>350</x>
                       <y>160</y>
                       <width>150</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="maxLength">
                      <number>20</number>
                     </property>
                    </widget>
                    <widget class="QLineEdit" name="lineEdit_2">
                     <property name="geometry">
                      <rect>
                       <x>350</x>
                       <y>100</y>
                       <width>150</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="maxLength">
                      <number>20</number>
                     </property>
                    </widget>
                    <widget class="QLineEdit" name="lineEdit">
                     <property name="geometry">
                      <rect>
                       <x>350</x>
                       <y>40</y>
                       <width>150</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="maxLength">
                      <number>8</number>
                     </property>
                    </widget>
                    <widget class="QLabel" name="label">
                     <property name="geometry">
                      <rect>
                       <x>290</x>
                       <y>40</y>
                       <width>21</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>ID</string>
                     </property>
                    </widget>
                    <widget class="QLabel" name="label_2">
                     <property name="geometry">
                      <rect>
                       <x>290</x>
                       <y>100</y>
                       <width>31</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>Nom</string>
                     </property>
                    </widget>
                    <widget class="QLabel" name="label_3">
                     <property name="geometry">
                      <rect>
                       <x>290</x>
                       <y>160</y>
                       <width>41</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>Prenom</string>
                     </property>
                    </widget>
                    <widget class="QLabel" name="label_4">
                     <property name="geometry">
                      <rect>
                       <x>290</x>
                       <y>220</y>
                       <width>21</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>Post</string>
                     </property>
                    </widget>
                    <widget class="QComboBox" name="comboBox_3">
                     <property name="geometry">
                      <rect>
                       <x>350</x>
                       <y>222</y>
                       <width>151</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);</string>
                     </property>
                     <item>
                      <property name="text">
                       <string>Ingenieur</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Ouvrier</string>
                      </property>
                     </item>
                    </widget>
                   </widget>
                   <widget class="QWidget" name="tab_2">
                    <attribute name="title">
                     <string>Show</string>
                    </attribute>
                    <widget class="QTableView" name="test">
                     <property name="geometry">
                      <rect>
                       <x>30</x>
                       <y>30</y>
                       <width>931</width>
                       <height>321</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                     </property>
                    </widget>
                   </widget>
                   <widget class="QWidget" name="tab_3">
                    <attribute name="title">
                     <string>Update</string>
                    </attribute>
                    <widget class="QLabel" name="label_5">
                     <property name="geometry">
                      <rect>
                       <x>250</x>
                       <y>60</y>
                       <width>21</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>ID</string>
                     </property>
                    </widget>
                    <widget class="QLabel" name="label_6">
                     <property name="geometry">
                      <rect>
                       <x>250</x>
                       <y>100</y>
                       <width>31</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>Nom</string>
                     </property>
                    </widget>
                    <widget class="QLabel" name="label_7">
                     <property name="geometry">
                      <rect>
                       <x>250</x>
                       <y>150</y>
                       <width>41</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>Prenom</string>
                     </property>
                    </widget>
                    <widget class="QLabel" name="label_8">
                     <property name="geometry">
                      <rect>
                       <x>250</x>
                       <y>200</y>
                       <width>21</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>Post</string>
                     </property>
                    </widget>
                    <widget class="QLineEdit" name="lineEdit_6">
                     <property name="geometry">
                      <rect>
                       <x>320</x>
                       <y>100</y>
                       <width>150</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="maxLength">
                      <number>20</number>
                     </property>
                    </widget>
                    <widget class="QLineEdit" name="lineEdit_7">
                     <property name="geometry">
                      <rect>
                       <x>320</x>
                       <y>150</y>
                       <width>150</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="maxLength">
                      <number>20</number>
                     </property>
                    </widget>
                    <widget class="QPushButton" name="pushButton_3">
                     <property name="geometry">
                      <rect>
                       <x>600</x>
                       <y>230</y>
                       <width>81</width>
                       <height>31</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>UPDATE</string>
                     </property>
                    </widget>
                    <widget class="QComboBox" name="comboBox_2">
                     <property name="geometry">
                      <rect>
                       <x>320</x>
                       <y>60</y>
                       <width>151</width>
                       <height>21</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);</string>
                     </property>
                    </widget>
                    <widget class="QComboBox" name="comboBox_4">
                     <property name="geometry">
                      <rect>
                       <x>320</x>
                       <y>200</y>
                       <width>151</width>
                       <height>20</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);</string>
                     </property>
                     <item>
                      <property name="text">
                       <string>Ingenieur</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Ouvrier</string>
                      </property>
                     </item>
                    </widget>
                   </widget>
                   <widget class="QWidget" name="tab_4">
                    <attribute name="title">
                     <string>Delete</string>
                    </attribute>
                    <widget class="QComboBox" name="comboBox">
                     <property name="geometry">
                      <rect>
                       <x>340</x>
                       <y>50</y>
                       <width>231</width>
                       <height>31</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);</string>
                     </property>
                    </widget>
                    <widget class="QPushButton" name="pushButton_2">
                     <property name="geometry">
                      <rect>
                       <x>700</x>
                       <y>200</y>
                       <width>161</width>
                       <height>41</height>
                      </rect>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);</string>
                     </property>
                     <property name="text">
                      <string>DEL</string>
                     </property>
                    </widget>
                   </widget>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
